gradle-plugin/doc/src/main/asciidoc/inc/examples/_index.adoc

```diff
@@ -0,0 +1,16 @@
+= Examples
+
+Let's have a look at some code. The following examples will demonstrate all available configurations variants:
+
+include::_groovy.adoc[]
+
+include::_dockerfile.adoc[]
```

Comment:
@rohanKanojia this file seems to be missing

Reply:
Let me remove this in a followup PR. I had moved this file to getting started scenarios

https://github.com/eclipse/jkube/blob/master/jkube-kit/doc/src/main/asciidoc/inc/getting-started/_dockerfile.adoc